experimental/overhead_matching/swag/data/satellite_embedding_database.py

```diff
@@ -0,0 +1,26 @@
+import common.torch.load_torch_deps
+import torch
+
+import experimental.overhead_matching.swag.data.vigor_dataset as vig_dataset
+
+def build_satellite_embedding_database(model: torch.nn.Module,
+                                       dataset: vig_dataset.VigorDataset, 
+                                       dataloader_kwards: dict | None = None,
```

Comment:
s/kwards/kwargs

Reply:
Should this function just take in a dataloader? There doesn't really seem to be a reason that this function needs access to the underlying dataset.